performance-tests/perf-tests.sh

```diff
@@ -253,21 +71,31 @@ onExit() {
   done
 }
 
+mapfile -t TESTCASES -d " " < <( find "${PERF_TESTS_DIR}" -type d -regex '.*/'"${TEST}" | sort )
+if [ ${#TESTCASES[@]} -eq 0 ]; then
+  echo -e "${RED}No test cases matched: find \"${PERF_TESTS_DIR}\" -type d -regex '.*/'\"${TEST}\" exiting ${NO_COLOR}" >&2
+  exit 1
+else
+  echo -e "${GREEN}Found tests: ${NO_COLOR}"
+  for test_path in "${TESTCASES[@]}"; do
+    echo "${test_path}"
+  done
+fi
+
 trap onExit EXIT
 
 TMP=$(mktemp -d)
 ON_SHUTDOWN+=("rm -rf ${TMP}")
 
 # Bring up Kafka
-ON_SHUTDOWN+=("runDockerCompose down")
+#ON_SHUTDOWN+=("runDockerCompose down")
 
-[[ -n ${PULL_CONTAINERS} ]] && runDockerCompose pull
+[[ -n ${PULL_CONTAINERS:-} ]] && runDockerCompose pull
 
 if [ "$(uname)" != 'Darwin' ]; then
-  # Async profiler not supported on macOS
+  # Kernel/perf-events profiler not supported on macOS
```

Comment:
I haven't tested running the perf tests on MacOS, but async profiler supports it these days.